lib/components/atoms/Anchor/Anchor.stories.mdx

```diff
@@ -0,0 +1,21 @@
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
+import Anchor, { AnchorVanilla } from '.';
+import { defaultAnchor } from './Anchor.mock';
+
+<Meta title="AnchorVanilla" component={AnchorVanilla} />
```

Comment:
There is a Module Props from @storybook/addon-docs which can render prop types. Can you also apply it?